doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -84,8 +81,6 @@ The OpenSSL FIPS provider supports these operations and algorithms:
 
 =item 3DES, see L<EVP_CIPHER-DES(7)>
 
-This is an unapproved algorithm.
-
```

Comment:
Why remove this?

Reply:
This has a indicator for the sign, but the verify part is still allowed.